### DISCUSSION TITLE
SSC user - change group (for additional permissions)?

### DISCUSSION THREAD

**Discussion by dragoonzwei** [Pryaxis/TShock#2263] [Q&A] [answered]
Is it possible to change the group of an SSC user?  I've tried according to the documentation, but I either get Syntax error, or when using quotes around name (my character name has a space), I get "user does not exist".

I tested by making a new character via the console, added them to my mods group (which has permissions for NPCs, so I can change housing setup), and it works no problem.  The test user has no space in their name.

Maybe I should add NPC permissions to the default group?  I'm not sure...

**Comment by Kojirremer** [comment, marked as answer]
Are you sure:

- The user really exists? Some configs let a user move around fairly unrestricted even without an account.
- You really typed the username *and* the group correctly? Not just quotation mark for spaces, upper and lower case is also important.
- You are not mixing the order of the command? The username goes first, and the group second.

An example of the command you are typing would be nice to have as well.

**Comment by dragoonzwei** [comment]
Well, in my testing to gather logs for your review, since I had no reason to be logged into the game, as I'd be on my other computer in the TShock console, I logged out of the game.  Then I tried moving my account to another group.

It worked on the first try... (;-_-)  I've always been logged in during my attempt to move my account to another group, using the server console.

Oh well, so long as it works.  I appreciate your help.  It did change my behavior in addressing the problem, which led to a resolution.

My next problem was being unable to perform basic tasks, since I neglected to give my mods group "default" permissions.  I modded the SQL Lite DB (backed up of course) to include those permissions, and all is well.

My testing steps are in the attached image.

![TShock-User-Group-Testing](https://user-images.githubusercontent.com/81780811/113455102-cf4b4380-93d7-11eb-9e30-1f74506fbca5.png)

**Reply by Kojirremer** [reply]
Hey, as long as it works, all's well.
